common/time/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libra-time"
+version = "0.1.0"
+authors = ["Libra Association <opensource@libra.org>"]
+description = "Libra libra-time"
+repository = "https://github.com/libra/libra"
+homepage = "https://libra.org"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0" }
```

Comment:
what's that?

Reply:
Required by the tests, I had [one failing](https://app.circleci.com/pipelines/github/libra/libra/27486/workflows/621fcfff-a926-4715-9f00-be744f74e013/jobs/179890) because it wasn't there and it's in all the common crates... No idea why.

Reply:
oh I think it's a hack to speed up rust-analyzer, wel let's leave it then :)